corehq/apps/userreports/templates/userreports/configurable_report.html

```diff
@@ -98,18 +98,20 @@
         The report will refresh automatically when the data is finished being populated.
         {% endblocktrans %}
     </div>
-    {% if request|toggle_enabled:"USER_CONFIGURABLE_REPORTS" and not report.spec.report_meta.created_by_builder %}
+    {% if request|toggle_enabled:"USER_CONFIGURABLE_REPORTS" and not report.spec.report_meta.created_by_builder or request|toggle_enabled:"REPORT_BUILDER" and report.spec.report_meta.created_by_builder %}
+    <div class="clearfix">
         <div class="pull-right">
-            <a class="btn btn-primary" href="
-                {% url 'edit_configurable_report' domain report.report_config_id %}
-            ">{% trans "Edit Report" %}</a>
-        </div>
-    {% elif request|toggle_enabled:"REPORT_BUILDER" and report.spec.report_meta.created_by_builder %}
-        <div class="pull-right">
-            <a class="btn btn-primary pull-right" href="
-                {% url 'edit_report_in_builder' domain report.report_config_id %}
-            ">{% trans "Edit Report" %}</a>
+            <a class="btn btn-primary" style="margin-bottom: 8px" href="
```

Comment:
I would have preferred to use some site-wide css class for adding this margin (I chose 8px arbitrarily), but there isn't such a class that I know of. Is there a best practice for this sort of thing?